python/rikai/types/geometry.py

```diff
@@ -392,6 +395,25 @@ def iou(
             return iou_arr[0]
         return iou_arr
 
+    def with_label(
+        self, text: str, color: str = get_option(CONF_RIKAI_VIZ_COLOR)
+    ):
+        """Spawn a list with current box and a `python.rikai.viz.Text`
```

Comment:
Lets rephrase the summary of a function to what is doing, instead of what it returns?